test/ejabberd_tests/tests/rest_SUITE.erl

```diff
@@ -183,6 +185,53 @@ messages_are_archived(Config) ->
         BobJID = maps:get(sender, Previous)
     end).
 
+send_and_wait(AliceJID, BobJID, Msg) ->
+    M = #{caller => BobJID, to => AliceJID, msg => Msg},
+    {?OK, _} = post(<<"/messages">>, M),
+    M1 = #{caller => AliceJID, to => BobJID, msg => Msg},
+    {?OK, _} = post(<<"/messages">>, M1),
+    timer:sleep(2000).
```

Comment:
How about bootstraping the archive in the same or similar way it was done in mam_SUITE? Take a look at [this helper](https://github.com/esl/MongooseIM/blob/735991785d1832e3273f5b3e98589bbf9d0dd66e/test/ejabberd_tests/tests/mam_helper.erl#L906) and how it was used.